sdk/python/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = 'symbol-sdk-python'
+version = '3.0.2'
+description = 'Symbol SDK'
+authors = ['Symbol Contributors <contributors@symbol.dev>']
+maintainers = ['Symbol Contributors <contributors@symbol.dev>']
+license = 'MIT'
+
+readme = 'README.md'
+
+packages = [{ include = 'symbolchain' }]
+
+repository = 'https://github.com/symbol/symbol/tree/main/sdk/python'
+
+keywords = ['symbol', 'sdk', 'sdk-core', 'Symbol SDK', 'Symbol SDK core']
```

Comment:
not sure if we need "core" keywords

Reply:
probably _not_ (I'd throw them away)

only reason we have them is that we've been discussed 2-level sdks, where one is low - txes, signing etc
other is high and can interact with rest servers